components/formats-bsd/src/loci/formats/tiff/TiffSaver.java

```diff
@@ -419,6 +423,7 @@ private void writeImageIFD(IFD ifd, int no, byte[][] strips,
     int tilesPerColumn = (int) ifd.getTilesPerColumn();
     boolean interleaved = ifd.getPlanarConfiguration() == 1;
     boolean isTiled = ifd.isTiled();
+    long defaultByteCount = 0L;
 
     RandomAccessInputStream in = null;
```

Comment:
you may want to use ``ry-with-resources `` https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
to avoid finally and make the code easier to read.
Also ``stripBuf`` does not seem to be closed.